sds_data_manager/stacks/backup_bucket_stack.py

```diff
@@ -64,8 +58,8 @@ def __init__(
         # This is the S3 bucket used by upload_api_lambda
         backup_bucket = s3.Bucket(
             self,
-            f"BackupDataBucket-{sds_id}",
-            bucket_name=f"sds-data-{sds_id}",
+            "BackupDataBucket",
+            bucket_name=f"sds-data-{source_account}-backup",
```

Comment:
This will result in a bucket name with a long string of numbers, which isn't really ideal... I like that it indicates the source bucket, but I don't think this is going to be useful.

Reply:
I don't think bucket names really matter at all, usually, you just get a random hash, so we could just use the default even and not specify a name. Here I am just saying I think we want a unique name for the bucket and one that we can point back to where it is coming from would possibly be helpful.